.github/workflows/build-deploy.yml

```diff
@@ -14,6 +14,32 @@ on:
         default: "Manual Workflow Run"
 
 jobs:
+  Sonar-Check:
+    runs-on: ubuntu-latest
+    env:
+      SONAR_TOKEN: ${{ secrets.SONARCLOUD_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # This workflow will automatically create a dummy commit in your repo if the last commit in your repo is 50 days (default) ago.
```

Comment:
If we want to use this keepalive-workflow action, then we should add it as a separate job, neither together with Sonar nor the normal build but on a separate schedule, see https://github.com/marketplace/actions/keepalive-workflow#via-github-actions-for-github-actions-users

We should also do this for all our GitHub repos, not just this one. Alternatively we could decide it is not needed since we in the future will continuously update all our git repos and so no action should get automatically suspended after 60 days any longer.

What do you think?

Reply:
I had no idea that we need to add it as a separate job, let me remove it from Sonar.

Reply:
Should I remove it from the build-and-deploy job as well?

Reply:
Not yet